game-core/src/main/java/games/strategy/triplea/ui/AbstractUiContext.java

```diff
@@ -283,7 +283,7 @@ public void setLockMap(final boolean lockMap) {
     try {
       prefs.flush();
     } catch (final BackingStoreException ex) {
-      ex.printStackTrace();
+      ClientLogger.logQuietly("Failed to flush preferences: " + prefs.absolutePath(), ex);
```

Comment:
Previously we logged to stderr, and now we log to stdout.  If you look at similar exception handlers in this file, you'll see this change is consistent with the others.